docs/static/releasenotes.asciidoc

```diff
@@ -39,6 +40,29 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-8]]
+=== Logstash 6.8.8 Release Notes
+
+* [Doc] Add tips for troubleshooting a pipeline. https://github.com/elastic/logstash/pull/11545[#11545]
+* [Doc] Update to include verification mode switch. https://github.com/elastic/logstash/pull/11284[#11284]
+* [Doc] Update offline-plugins.asciidoc to use correct command syntax. https://github.com/elastic/logstash/pull/10912[#10912]
+* [Doc] Update logging.asciidoc to emphasize that logging to console is included in out-of-the-box settings. https://github.com/elastic/logstash/pull/10717[#10717]
+
+==== Plugins
+
+*Memcached Filter*
+
+* Docs: Fixed link formatting issues in doc https://github.com/logstash-plugins/logstash-filter-memcached/pull/16[#16]
```

Comment:
We can safely delete this one.  It fixed formatting that caused doc builds to break, so nothing a user would care about.

Reply:
I'm minded to keep this in as the version of the memcached filter changed since the last release, and having a reason in the changelog may avoid user confusion.

Reply:
Makes since from that standpoint. "Nothing to see here, folks!"  :-)